addons/core/translations/en-us.yaml

```diff
@@ -15,6 +15,7 @@ titles:
   home: Home
   iam: IAM
   new: New
+  pki: New PKI Worker
```

Comment:
I think this probably should live in the specific resource file instead of the root translation?

Reply:
@ZedLi  fixed